### DISCUSSION TITLE
Exception: No such proxy method: dlg. On Mediareceived(System. String)

### DISCUSSION THREAD

**Discussion by lieyou003** [yasirkula/UnityNativeGallery#324] [Q&A]
Unity 2022.3.33f1  Error message when using GetImageFromGallery method in Android package   
Exception: No such proxy method: dlg. On Mediareceived(System. String)

**Comment by yasirkula** [comment]
Are you using NativeGallery's latest version? There was a fix specifically for this problem: https://github.com/yasirkula/UnityNativeGallery/releases/tag/v1.8.0

**Reply by lieyou003** [reply]
Thank you, we are using the latest version. I found the reason, there was code confusion during the packaging process, so an error was reported

**Reply by yasirkula** [reply]
So the issue is resolved then? Do you know what triggered that confusion during the packaging process; maybe a code obfuscator or something like that?
